GymBuddies/APIManagers/ParseAPIManager.m

```diff
@@ -44,4 +44,92 @@ +(void)logOut:(ParseManagerLogOutCompletionBlock) completion{
 }
 
 
++(void)fetchBodyZones:(ParseManagerFetchingDataRowsCompletionBlock) completion{
+    PFQuery *query = [PFQuery queryWithClassName:@"BodyZone"];
+    
+    ParseManagerFetchingDataRowsCompletionBlock block = ^void(NSArray *elements, NSError *error){
+        completion(elements, error);
+    };
+    
+    [query findObjectsInBackgroundWithBlock:block];
+}
+
+
++ (Exercise *)createExercise:(Exercise *)exercise
+           completion:(ParseManagerCreateCompletionBlock) completion {
+    
+    Exercise *newExercise = [Exercise initWithAttributes:exercise.title caption:exercise.caption author:exercise.author video:exercise.video image:exercise.image bodyZoneTag:exercise.bodyZoneTag];
+    
+    ParseManagerCreateCompletionBlock block = ^void(BOOL succeeded, NSError * _Nullable error){
+        completion(succeeded, error);
+        if(!succeeded){
+            return;
+        }
+    };
+    
+    [newExercise saveInBackgroundWithBlock:block];
+
+    [self saveExercise:newExercise completion:block];
+
+    return newExercise;
+}
+
+
++ (void)saveExercise:(Exercise *)exercise completion:(ParseManagerCreateCompletionBlock)completion{
+    PFObject *savedExercise = [PFObject objectWithClassName:@"SavedExercise"];
```

Comment:
make "SavedExercise" a constant and assign it to a static variable (avoid string literals).